src/Conf.ml

```diff
@@ -231,6 +234,14 @@ module Formatting = struct
     C.choice ~names ~all ~env ~doc ~allow_inline:true ~update:(fun conf x ->
         {conf with break_string_literals= x} )
 
+  let case_style =
+    let doc = "TODO." in
+    let env = Arg.env_var "OCAMLFORMAT_CASE_STYLE" in
+    let names = ["case-style"] in
```

Comment:
`--no-indicate-nested-or-patterns` ? Note that `--sparse` already does this, but also includes other formatting changes so that (bodies and when guards of) nested or-patterns are not hidden.